app/start_handlers.py

```diff
@@ -242,7 +200,23 @@ async def post(self, request):
             else:
                 raise ex
 
-        await remember(uac_json['caseId'], request)
+        logger.info('logging uac_json', uac_json=uac_json)
+
+        if uac_json['caseId'] is None:
+            logger.info('unlinked case', client_ip=request['client_ip'])
+            session = await get_session(request)
+            session['attributes'] = {}
+            session['case'] = uac_json
+            if data.get('adlocation'):
+                session['adlocation'] = data.get('adlocation')
+            await remember(str(uuid.uuid4()), request)
+            raise HTTPFound(request.app.router['CommonEnterAddress:get'].url_for(
+                display_region=display_region,
+                user_journey='start',
+                sub_user_journey='unlinked'
+            ))
+        else:
+            await remember(uac_json['caseId'], request)
```

Comment:
I know this is not changed, but does the comment a few lines below beginning "SOMEHOW NEED TO MAP ADDRESS ... "  represent unfinished work ?

Reply:
No, its a comment from before my time - it has been done.
I'll remove the comment